src/js/common/containers/MHVApp.jsx

```diff
@@ -42,7 +42,9 @@ export class MHVApp extends React.Component {
 
   needsTermsAcceptance = () => this.props.account.state === 'needs_terms_acceptance';
 
-  isAccessible = () => ['existing', 'upgraded'].includes(this.props.account.state);
+  hasAccount = () => ['existing', 'upgraded'].includes(this.props.account.state);
```

Comment:
If we get rid of the `mhv_account_state` stuff altogether, we could (1) base eligibility for account creation in `MHVApp` on the existence of `mhvac` in the services list and (2) determine existence of an account from the MHV account level (`nil` for no account, otherwise string value) if it gets exposed in the user object.

Reply:
Maybe mhv_account_state could eventually be equivalent to the account type, then we wouldn't need to track it elsewhere, seeing as we have a pretty reliable means of determining it now. But yeah, I agree and would prefer not to persist anything to postgres eventually that isn't just a log of what action vets.gov has taken, such as creating or upgrading.